R/PlatinumGenomes-QC/Sample-Level-QC.md

```diff
@@ -565,67 +565,106 @@ result <- DisplayAndDispatchQuery("./sql/homozygous-variants.sql",
 
 ```
 # Compute the expected and observed homozygosity rate for each individual.
+# FROM DOCUMENTATION IN VCFTOOLS
```

Comment:
Maybe adding the following link would help folks go to the specific line in the code that this references:

https://sourceforge.net/p/varianttools/git/ci/master/tree/variant_tools/utils.cpp#l326